src/Dropdown/Dropdown.js

```diff
@@ -327,7 +329,7 @@ class Dropdown extends Util.mixin(BindMixin, KeyDownMixin) {
           {this.getMenuItems(props.items)}
         </ul>
       );
-      let menuPosition = Object.assign({}, state.menuPosition);
+      let dropdownMenuWrapperStyle = Object.assign({}, state.menuPosition);
```

Comment:
I changed this variable name to be more explicit about what is actually being set.